pybamm_tea/tea.py

```diff
@@ -333,209 +333,102 @@ def print_stack_breakdown(self):
-            {"Parameter": "Stoichiometries", "Unit": "", "Value": ""},
-            {
-                "Parameter": "Negative electrode stoichiometry at 0% SoC",
-                "Unit": "-",
-                "Value": stack_ed.get("Negative electrode stoichiometry at 0% SoC"),
-            },
-            {
-                "Parameter": "Negative electrode stoichiometry at 100% SoC",
-                "Unit": "-",
-                "Value": stack_ed.get("Negative electrode stoichiometry at 100% SoC"),
-            },
-            {
-                "Parameter": "Positive electrode stoichiometry at 100% SoC",
-                "Unit": "-",
-                "Value": stack_ed.get("Positive electrode stoichiometry at 100% SoC"),
-            },
-            {
-                "Parameter": "Positive electrode stoichiometry at 0% SoC",
-                "Unit": "-",
-                "Value": stack_ed.get("Positive electrode stoichiometry at 0% SoC"),
-            },
+
+        parameters = [
+            "Stack potentials:",
+            "Stack average OCP [V]",
+            "Minimal OCP [V]",
+            "Maximal OCP [V]",
+            "Electrode potentials:",
+            "Negative electrode average OCP [V]",
+            "Positive electrode average OCP [V]",
+            "n/p ratio's:",
+            "Practical n/p ratio",
+            "Theoretical n/p ratio",
+            "Stack capacities:",
+            "Volumetric stack capacity [A.h.L-1]",
+            "Gravimetric stack capacity [A.h.kg-1]",
+            "Capacity [mA.h.cm-2]",
+            "Electrode capacities:",
+            "Negative electrode theoretical capacity [mA.h.cm-2]",
+            "Positive electrode theoretical capacity [mA.h.cm-2]",
+            "Negative electrode volumetric capacity [A.h.L-1]",
+            "Positive electrode volumetric capacity [A.h.L-1]",
+            "Negative electrode gravimetric capacity [A.h.kg-1]",
+            "Positive electrode gravimetric capacity [A.h.kg-1]",
+            "Active material capacities:",
+            "Negative electrode active material practical capacity [A.h.kg-1]",
+            "Positive electrode active material practical capacity [A.h.kg-1]",
+            "Negative electrode active material capacity [A.h.kg-1]",
+            "Positive electrode active material capacity [A.h.kg-1]",
+            "Stoichiometries:",
+            "Negative electrode stoichiometry at 0% SoC",
+            "Negative electrode stoichiometry at 100% SoC",
+            "Positive electrode stoichiometry at 100% SoC",
+            "Positive electrode stoichiometry at 0% SoC",
         ]
-        # Create the DataFrame from the list of dictionaries
+
+        names = []
+        units = []
+        values = []
+        for parameter in parameters:
+            if "[" in parameter:
+                name, unit = parameter.split("[")
+                names.append(name.strip())
+                units.append(unit.rstrip(']'))
+            else:
```

Comment:
why do you need this? I don't think you need to have the subheadings as part of the dataframe